examples/stock/company.v.py

```diff
@@ -1,21 +1,23 @@
 # Own shares of a company!
 company: address
-avail_shares: num
-total_shares: num
-price: wei_value
+total_shares: currency_value
+price: num (wei / currency)
 
```

Comment:
I checked and public works the same as it does in solidity with variables (automatically creating getters).  I'd make `total_shares` and `price` public so the user can call `get_total_shares()` and `get_price()`.